src/makeCalculator.js

```diff
@@ -37,7 +37,26 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const obj = {
```

Comment:
```suggestion
  return {
```
you may return it immediately

Reply:
FYI